lib/upload.js

```diff
@@ -3546,7 +3546,6 @@
     var autoProcessQueue = element.hasAttribute(dataAutoprocess) && element.getAttribute(dataAutoprocess) !== 'false';
     var size = element.hasAttribute(dataMaxSize) ? element.getAttribute(dataMaxSize) : 2000000;
     var acceptedFiles = element.hasAttribute(dataAllowedTypes) ? element.getAttribute(dataAllowedTypes) : null;
-    var maxFilesize = vl.util.bytesToSize(size, false, 1000);
```

Comment:
Deze is/was dan stuk of deed de omzetting niet zoals verwacht?

Reply:
als ik het wat begrijp is dit een breaking change en heeft het impact op iedereen die dat gebruikte.
Of werkte het voorheen niet correct en nu wel?

Reply:
Dit deed een vertaling van de size in bytes naar een size zonder eenheid. Het ging er, zoals je later in de code merkt vanuit dat het MB zou zijn, maar dat is niet altijd het geval. Bovendien is de base hier 1000 ipv 1024. Voor de errormessage wordt wel 1024 gedaan. 
Beter leek me om de maxFileSize in bytes te houden en daarna ook met de filesize in bytes te vergelijken om te zien of het bestand mag geupload worden of niet.

Reply:
Merk op, dat ik ondertussen ook de 2000000 (2 lijnen hoger) aangepast hebt, zodat dit consistent is met de documentatie.

Reply:
lijkt me ok. Ik zou ook overal consistent dezelfde eenheid gebruiken met bytes als voorkeur. Geeft dan genoeg precisie voor wie het gebruikt en het lijkt ook de standaard in andere frameworks